### ISSUE TITLE
[mongodb-sharded] Lint failed when increase `arbiter.replicas`

### ISSUE THREAD

**Issue by PhanSon95** [bitnami/charts#4781]
**Which chart**:
Chart: mongodb-sharded
Version: 3.2.4

**Describe the bug**
When increase number of replicas arbiter `0->1`. Lint chart will fail

**To Reproduce**

1. Fetch chart bitnami/mongodb-sharded version 3.2.4
```
helm fetch bitnami/mongodb-sharded --version 3.2.4 --untar
```
2. Change values replicas arbiter from 0 => 1 at [here](https://github.com/bitnami/charts/blob/master/bitnami/mongodb-sharded/values.yaml#L241)
3. Lint chart 
```
helm lint -f values.yaml .
```
**Expected behavior**

**Version of Helm and Kubernetes**: 
Helm 3.4.1
Kubernetes: 1.17.7

**Additional context**

**Comment by rafariossaa** [comment]
Hi,
  Could you try running `helm dependency update` before the `lint` ? I think `lint` is not aware of chart dependencies.

**Comment by paidaki** [comment]
Hello, even installing with `arbiter.replicas` greater than `0` causes an error.

`helm install --namespace=mongodb --timeout=10m0s --values=/home/shell/helm/values-mongodb-sharded-3.2.5.yaml --version=3.2.5 --wait=true mongodb /home/shell/helm/mongodb-sharded-3.2.5.tgz`

`Error: template: mongodb-sharded/templates/shard/shard-arbiter-statefulset.yaml:72:19: executing "mongodb-sharded/templates/shard/shard-arbiter-statefulset.yaml" at <include "mongodb-sharded.name" $>: error calling include: template: no template "mongodb-sharded.name" associated with template "gotpl"`

**Comment by SoloSynth1** [comment]
Hi,
This issue is caused by the flow control in `templates/shard/shard-arbiter-statefulset.yaml` evaluating a value `mongodb-sharded.name` which should be present when creating the arbiter container, but it is not. You can add it back in `templates/_helpers.tpl`:

#### templates/_helpers.tpl
```
{{/*
Expand the name of the chart.
*/}}
{{- define "mongodb-sharded.name" -}}
{{- include "common.names.name" . -}}
{{- end -}}
```

This is included in the PR I just linked to this issue (#4812). Thanks.

**Comment by rafariossaa** [comment]
Ok, let's wait until #4812 is merged.
